packages/frontend-shared/cypress/support/mock-graphql/mountFragment.ts

```diff
@@ -150,7 +150,7 @@ export const registerMountFn = ({ plugins }: MountFnOptions = {}) => {
     const definition = document.definitions[0]
 
     if (definition.kind === 'OperationDefinition' && definition.name) {
-      mutationResolvers[definition.name.value] = resolver
+      mutationResolvers.set(definition.name.value, resolver)
```

Comment:
Mutation resolvers is a Map, and we use `.clear()` to reset the Map between each test. We need to use `.get()` and `.set()` in conjunction with `.clear()`, otherwise we don't [clean up appropriately between tests](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map#setting_object_properties).

Reply:
Nice catch